templates/user/settings/applications.tmpl

```diff
@@ -55,6 +55,12 @@
 </div>
 
 <div class="ui small basic delete modal" id="delete-token">
+	{{/*
+		The wording here is a bit confusing.
+		The action asks for deleting an access token.
+		Confirming is therefore the destructive option and
+		should be deemphasized because it cannot be undone.
+	*/}}
```

Comment:
Not sure if we leave comments like this in the code. Haven't seen it before.

Reply:
I was on the fence. But then looking at the template below where „yes” was in `red` and `no` was marked up I thought it warranted an explanation.

Now I don't want to make contributors hunt the code history to find the motivation. And the comment wouldn't get rendered out (I checked). Aside, Go files have comments for functions.

I don't mind if I shall remove it.

Reply:
Tbh would be better if the comment wouldn't be rendered to the client(if anyone wants to find this motivation, opting to go search it in the code should be de-facto default).

Reply:
I don't think the template comments are rendered?

Reply:
They aren't. I double checked.